src/frontends/paddle/src/op/strided_slice.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <limits.h>
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+Output<Node> idx_node(const std::string& tensor_alias,
+                      const std::string& list_alias,
+                      const std::string& attr_alias,
+                      const NodeContext& node) {
+    if (node.has_input(tensor_alias)) {
+        return node.get_input(tensor_alias);
+    } else if (node.has_input(list_alias)) {
+        auto inputs = node.get_ng_inputs(list_alias);
+        return std::make_shared<default_opset::Concat>(inputs, 0);
+    } else {
+        auto values = node.get_attribute<std::vector<int32_t>>(attr_alias);
+        return default_opset::Constant::create(element::i32, {values.size()}, values);
+    }
+}
+
+NamedOutputs strided_slice(const NodeContext& node) {
```

Comment:
It looks paddle "stridedslice" is quite similar to "slice". Can you merge them?

And there is a bug to "slice" op converter in existing master branch. Please compare it to https://github.com/openvinotoolkit/openvino/blob/4b6c898a6f67815f8e0b6f5afb36903cc6607553/src/frontends/paddle/src/op/slice.cpp.

Reply:
merged into one